extension/src/popup/components/accountHistory/HistoryItem/index.tsx

```diff
@@ -40,7 +44,12 @@ export const HistoryItem = ({
     type,
     transaction_attr: { operation_count: operationCount },
     isPayment = false,
+    isSwap = false,
   } = operation;
+  let sourceAssetCode;
+  if ("source_asset_code" in operation) {
+    sourceAssetCode = operation.source_asset_code;
+  }
```

Comment:
hm, is there a danger in just setting `const sourceAssetCode = operation.source_asset_code;`?

Reply:
kind of annoying of typescript imo but 🤷